test/functional/tests/data_integrity/test_data_integrity_long.py

```diff
@@ -82,12 +82,12 @@ def test_data_integrity_5d_dss(filesystems):
             sync()
 
     with TestRun.step("Run test workloads on filesystems with verification"):
-        fio_run = Fio().create_command()
+        fio = Fio()
+        fio_run = fio.create_command()
+        fio.base_cmd_parameters.set_param('alloc-size', Unit.MebiByte.value)
```

Comment:
I don't like this way of passing the value - alloc-size is given in kb, and this led me to believe that the alloc-size was 1 MiB, when in fact it's 1GiB
Maybe use Size(1, GiB).get_value(KiB)?